typescript-json-schema.ts

```diff
@@ -1088,7 +1088,13 @@ export class JsonSchemaGenerator {
                         const targetName: string = (<any>clazzType).mapper?.target?.value;
                         const indexedAccessType = <ts.IndexedAccessType>typ;
                         const symbols: Map<string, ts.Symbol> = (<any>indexedAccessType.objectType).members;
-                        const targetSymbol = symbols?.get(targetName);
+                        let targetSymbol = symbols?.get(targetName);
+
+                        // it looks like a bug in typescript,
+                        // if property start with 2 underscores and more, in objectType.members, have a 1 unnecessary underscore
```

Comment:
Can you link to an issue in the ts repo so we know when we can remove this hack?

Reply:
@domoritz https://github.com/microsoft/TypeScript/issues/54704

Reply:
Link from the comment I mean

Reply:
@domoritz so you want me to change the comment in the code, and you will merge this PR?

Reply:
@domoritz so, I switched to use in utility of typescript to convert to `escape` string